src/masternode-payments.cpp

```diff
@@ -777,6 +779,62 @@ bool CMasternodePayments::ProcessBlock(int nBlockHeight, CConnman& connman)
     return false;
 }
 
+void CMasternodePayments::CheckPreviousBlockVotes(int nPrevBlockHeight)
+{
+    LogPrint("mnpayments", "CMasternodePayments::CheckPreviousBlockVotes -- nPrevBlockHeight=%d, expected voting MNs:\n",
```

Comment:
This function should not be run until winners list is synced (no data synced yet, no need to spam with `no vote received`), should have
```
if (!masternodeSync.IsWinnersListSynced()) return;
```
at the very start of it.

Reply:
fixed